packages/turbo/bin/turbo

```diff
@@ -1,10 +1,257 @@
 #!/usr/bin/env node
 
-const { generateBinPath } = require("../node-platform");
+/**
+ * We need to run a platform-specific `turbo`. The dependency _should_
+ * have already been installed, but it's possible that it has not.
+ */
 
+const child_process = require('child_process');
+const fs = require('fs');
+const path = require('path');
+
+// If we do not find the correct platform binary, should we attempt to install it?
+const SHOULD_INSTALL = true;
+
+// If we do not find the correct platform binary, should we trust calling an emulated variant?
+const SHOULD_ATTEMPT_EMULATED = true;
```

Comment:
I left these in as configuration variables; but we can actually hardcode the decisions here.